audit/k8s-cip-test-prod/services/enabled.json

```diff
@@ -114,6 +117,122 @@
     "parent": "projects/693665670941",
     "state": "ENABLED"
   },
+  {
+    "config": {
+      "authentication": {},
+      "documentation": {
+        "summary": "A data platform for customers to create, manage, share and query data."
```

Comment:
For services in particular, but also in general, I wonder if we should filter this through some sort of simplication filters?  services: I think we only care about the list of enabled service names.  In other places, we likely can filter out etags etc?